sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -253,6 +253,8 @@ void nav_init_stage(void)
   }
 }
 
+uint32_t stage_timer_msec = 0;
+
```

Comment:
you can add local variables directly to the flight plan. See: https://github.com/paparazzi/paparazzi/blob/master/conf/flight_plans/ENAC/fish_outdoor.xml#L44